src/main/java/org/folio/circulationbff/service/impl/SearchServiceImpl.java

```diff
@@ -294,6 +316,20 @@ private static BffSearchItem buildBffSearchItem(ItemContext itemContext, SearchI
       .volume(item.getVolume())
       .inTransitDestinationServicePointId(toUUID(item.getInTransitDestinationServicePointId()));
 
+    var permanentLoanTypeId = item.getPermanentLoanTypeId();
+    if (permanentLoanTypeId != null) {
+      bffSearchItem.permanentLoanType(new BffSearchItemPermanentLoanType()
+        .id(item.getPermanentLoanTypeId())
+        .name(itemContext.permanentLoanType.getName()));
```

Comment:
Beware of NPE: loan type in context might be null

```suggestion
        .name(itemContext.permanentLoanType().getName()));
```

Reply:
updated